<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FontToBytes</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../assets.qrc">
    <normaloff>:/appIcon</normaloff>:/appIcon</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="sideBarWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>12</number>
       </property>
       <property name="topMargin">
        <number>24</number>
       </property>
       <property name="rightMargin">
        <number>12</number>
       </property>
       <property name="bottomMargin">
        <number>24</number>
       </property>
       <item>
        <widget class="QGroupBox" name="fontSizeGroupBox">
         <property name="title">
          <string>Font Size in px</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="1" column="0">
           <widget class="QLabel" name="widthLabel_2">
            <property name="text">
             <string>Width</string>
            </property>
            <property name="buddy">
             <cstring>widthLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="widthLineEdit">
            <property name="placeholderText">
             <string>1-255px</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="heightLineEdit">
            <property name="placeholderText">
             <string>1-255px</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="heightLabel_2">
            <property name="text">
             <string>Height</string>
            </property>
            <property name="buddy">
             <cstring>heightLineEdit</cstring>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="readingModeGroupBox">
         <property name="title">
          <string>Reading Mode</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="topBottomRadioButton">
            <property name="text">
             <string>Top-Bottom</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="leftRightRadioButton">
            <property name="text">
             <string>&amp;Left-Right</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="miscOptionsGroupBox">
         <property name="title">
          <string>Misc Options</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QCheckBox" name="invertBitsCheckBox">
            <property name="text">
             <string>Invert Bits</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="bitNumberingCheckBox">
            <property name="text">
             <string>Reverse Bits (MSB)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Output Format</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QComboBox" name="formatComboBox"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="palette">
          <palette>
           <active>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>128</red>
               <green>128</green>
               <blue>128</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Link">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>128</red>
               <green>128</green>
               <blue>128</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="LinkVisited">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>128</red>
               <green>128</green>
               <blue>128</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>128</red>
               <green>128</green>
               <blue>128</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Link">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>128</red>
               <green>128</green>
               <blue>128</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="LinkVisited">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>128</red>
               <green>128</green>
               <blue>128</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>128</red>
               <green>128</green>
               <blue>128</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Link">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>128</red>
               <green>128</green>
               <blue>128</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="LinkVisited">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>128</red>
               <green>128</green>
               <blue>128</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Font to Bytes, Qt port.&lt;br/&gt;Â© 2019 Dominik Kapusta&lt;br/&gt;&lt;a href=&quot;http://kapusta.cc&quot; style=&quot;color: #7F7F7F;&quot;&gt;http://kapusta.cc&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="DroppableStackedWidget" name="stackedWidget">
      <property name="minimumSize">
       <size>
        <width>480</width>
        <height>302</height>
       </size>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="PromptLabel" name="infoLabel">
          <property name="text">
           <string>Start by setting up
font conversion
parameters</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="1">
         <widget class="DropLabel" name="promptLabel">
          <property name="text">
           <string>Drop your
PNG file here!</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTextBrowser" name="textBrowser">
          <property name="lineWrapMode">
           <enum>QTextEdit::NoWrap</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="copyButton">
            <property name="text">
             <string>Copy to Clipboard</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveAsButton">
            <property name="text">
             <string>Save As...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="ColoredStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>865</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DropLabel</class>
   <extends>QLabel</extends>
   <header>droplabel.h</header>
  </customwidget>
  <customwidget>
   <class>DroppableStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header>droppablestackedwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PromptLabel</class>
   <extends>QLabel</extends>
   <header>promptlabel.h</header>
  </customwidget>
  <customwidget>
   <class>ColoredStatusBar</class>
   <extends>QStatusBar</extends>
   <header>coloredstatusbar.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>heightLineEdit</tabstop>
  <tabstop>widthLineEdit</tabstop>
  <tabstop>topBottomRadioButton</tabstop>
  <tabstop>leftRightRadioButton</tabstop>
  <tabstop>invertBitsCheckBox</tabstop>
  <tabstop>bitNumberingCheckBox</tabstop>
  <tabstop>formatComboBox</tabstop>
  <tabstop>textBrowser</tabstop>
  <tabstop>copyButton</tabstop>
  <tabstop>saveAsButton</tabstop>
 </tabstops>
 <resources>
  <include location="../assets.qrc"/>
 </resources>
 <connections/>
</ui>
